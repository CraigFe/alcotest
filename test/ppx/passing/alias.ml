let _ = [%here]
let _ = [%check: int list]
