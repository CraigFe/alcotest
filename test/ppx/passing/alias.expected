let _ =
  { Lexing.pos_fname = "alias.ml"; pos_lnum = 1; pos_cnum = 8; pos_bol = 0 }
let _ =
  fun ?(here=
    {
      Lexing.pos_fname = "alias.ml";
      pos_lnum = 2;
      pos_cnum = 24;
      pos_bol = 16
    }) ->
    fun ?msg ->
      fun ~expected ->
        fun actual ->
          Ppx_alcotest_runtime.check ~here
            ~testable:(Alcotest.list Alcotest.int) ?msg ~expected actual
